src/snowflake/connector/s3_util.py

```diff
@@ -53,7 +53,9 @@ class SnowflakeS3Util:
     """S3 Utility class."""
 
     @staticmethod
-    def create_client(stage_info, use_accelerate_endpoint=False) -> Session.resource:
+    def create_client(
+        stage_info, use_accelerate_endpoint=False, use_s3_regional_url=False
```

Comment:
I don't know exactly what `stage_info` is could you please verify and add that too?
```suggestion
        stage_info, use_accelerate_endpoint: bool = False, use_s3_regional_url: bool = False
```